public/electron.js

```diff
@@ -82,15 +82,35 @@ function initialize(queues) {
   )
 
   mainWindow.webContents.on('did-finish-load', () => mainWindow.show())
-  // mainWindow.webContents.openDevTools()
+
+  // Open devtools if DEV_TOOLS is set and has a value, presumably 1
```

Comment:
If `PUBLISH50_DEV_TOOLS`. No assumptions about the value except that it has to be truthy (e.g., not empty string).